package/nix/profile.py

```diff
@@ -81,6 +81,6 @@ def mkColumn(c, w):
     return ' ' + c.ljust(w) + ' '
 
 columns = ['|'.join((mkColumn(c0, column_width0), mkColumn(c1, column_width1), mkColumn(c2, column_width2), mkColumn(c3, column_width3))) for (c0, c1, c2, c3) in final_table]
-columns[1] = columns[1].replace('|', '+').replace(' ', '-')
```

Comment:
Does this script run on one of our runners somewhere, or is it just here to generate manual reports?

Reply:
This script is used for performance tests on a dedicated runner of ours. We have [a mechanism in `haskell-backend` to trigger the performance tests](https://github.com/runtimeverification/haskell-backend/blob/master/.github/workflows/kevm-performance-test.yaml#L81) via git PR comments .